src/_includes/components/image_modal.njk

```diff
@@ -0,0 +1,58 @@
+{# Image modal content - Macro to be imported elsewhere
+Use like so:
+{% import "components/image_modal.njk" as image_modal with context %}
+{{ image_modal.modal(image_path, image_alt) }} #}
+{% macro modal(image_name, image_path, image_alt='') %}
+<div class="usa-modal usa-modal--lg"
+        id="image-modal-{{image_name}}"
+        aria-labelledby="image-modal-heading"
+        aria-describedby="image-modal-description">
+    <div class="usa-modal__content">
+        <div class="padding-x-4 padding-bottom-4">
+            {# This is <p>, instead of <h2>. This is so it doesn't break certain components like the in-page nav, or mess with ordering on alternate viewing options #}
+            <p class="usa-modal__heading" id="image-modal-heading">
+            Image
+            </p>
+            <img src="{{config.baseUrl}}{{image_path}}"
+                style="border: 1px solid #555;"
+                alt="{{ image_alt }}"/>
+            <div class="usa-prose">
+                <span id="image-modal-description">
+                    {% if image_alt %}
+                        {{ image_alt }}
+                    {% else %}
+                        {{ image_name }}
```

Comment:
Do we really want the file name there? I feel like it's better to have nothing, or maybe "Description pending" or something.

Reply:
I agree! A default other than the file name makes more sense. I put "Description pending" for now, but it's easily changed.